src/org/zaproxy/zap/extension/bugTracker/BugTrackerJiraIssue.java

```diff
@@ -0,0 +1,13 @@
+package org.zaproxy.zap.extension.bugTracker;
+
+public class BugTrackerJiraIssue {
+
+	public BugTrackerJiraIssue(RaiseManualIssueDialog dialog, int index) {
+		dialog.addTextField(index, "zest.dialog.script.label.title","One");
```

Comment:
You shouldnt be using "zest" labels here, especially as some users might not have Zest installed